python/publish.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python setup.py sdist upload -r pypi
```

Comment:
since there is a `Makefile` you could consider deleting this file and add a new target to it:
```make
publish: test
	python setup.py sdist upload -r pypi
```

Reply:
Done.